bundles/SimpleBackendSearchBundle/src/DependencyInjection/PimcoreSimpleBackendSearchExtension.php

```diff
@@ -0,0 +1,39 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\SimpleBackendSearchBundle\DependencyInjection;
+
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\Extension;
+use Symfony\Component\DependencyInjection\Loader\YamlFileLoader;
+
+class PimcoreSimpleBackendSearchExtension extends Extension
+{
+    public function load(array $configs, ContainerBuilder $container)
+    {
+        // on container build the shutdown handler shouldn't be called
+        // for details please see https://github.com/pimcore/pimcore/issues/4709
```

Comment:
is that really necessary for that bundle?

Reply:
removed since the CoreBundle already does the job. c9469399ee0ce23cac1f367032494d54adc43726